docs/development/build.md

```diff
@@ -29,34 +29,34 @@ For building the latest code in master, follow the latest version of this page
 [here](https://github.com/apache/druid/blob/master/docs/development/build.md):
 make sure it has `/master/` in the URL.
 
-#### Prerequisites
+## Prerequisites
 
-##### Installing Java and Maven
+### Installing Java and Maven
 
 - See our [Java documentation](../operations/java.md) for information about obtaining a supported JDK
 - [Maven version 3.x](http://maven.apache.org/download.cgi)
 
-##### Other dependencies
+### Other Dependencies
 
 - Distribution builds require Python 3.x and the `pyyaml` module.
 - Integration tests require `pyyaml` version 5.1 or later.
 
-##### Downloading the source
+## Downloading the Source Code
 
 ```bash
 git clone git@github.com:apache/druid.git
 cd druid
 ```
 
-#### Building from source
+## Building from Source
 
 The basic command to build Druid from source is:
 
 ```bash
 mvn clean install
 ```
 
-This will run static analysis, unit tests, compile classes, and package the projects into JARs. It will _not_ generate the source or binary distribution tarball.
+This will run static analysis, unit tests, compile classes, and package the projects into JARs. It will _not_ generate the source or binary distribution tarball. Take note that this build will take up to an hour to complete.
```

Comment:
Consider something like this, as the build and test time will only increase as we add more tests, extensions, etc:
```suggestion
This will run static analysis, unit tests, compile classes, and package the projects into JARs. It will _not_ generate the source or binary distribution tarball. Note that this build may take some time to complete.
```